example/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"github.com/ShaileshSurya/hammer"
+)
+
+func handleErr(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func main() {

```

Comment:
We can break the main into two different functions 
1. `ExampleForExecute()`
2. `ExampleForExecuteInto()`

As this is will be more visible to the library reader. We want to make our intent as much clear as possible. 

Or This can be broken into two different files altogether. One for `execute` and another for `executeInto`.

Reply:
Changed into two separate files.

examples/execute/main.go
example/execute_into/main.go